runtime/compiler/net/CommunicationStream.cpp

```diff
@@ -55,6 +58,11 @@ bool CommunicationStream::useSSL()
            compInfo->getJITServerSslRootCerts().size());
    }
 
+bool CommunicationStream::shouldReadRetry ()
+   {
+   return (_readRetryAttempt < MAX_READ_RETRY);
```

Comment:
Since this method is so small, it's better to move it's implementation in the hpp file

Reply:
Moved the method.